### PR TITLE
Networking inside containers

### PR DESCRIPTION
**Pull request by LK4D4**
ping @icecrime

### CODE REVIEW

**Comment by LK4D4** [comment]
ping @erikh

File: sandbox/controller.go [hunk 1]
```diff
@@ -35,7 +41,17 @@ func (c *Controller) New() (string, error) {
...
+	driver execdriver.Driver
+}
+
+func (s *NativeSandbox) AddIface(iface *execdriver.NetworkSettings) error {
+	log.Printf("Sandbox %v", s)
+	return s.driver.AddIface(s.ID, iface)
+}
+
 type Sandbox interface {
-	Exec(cmd string, args []string, env []string) error
```
Comment (erikh) [inline sandbox/controller.go:39]: can we not remove this and just comment it out? I think we're going to need it.

Comment (LK4D4) [inline sandbox/controller.go:39]: It is commented :)

Comment (erikh) [inline sandbox/controller.go:39]: oh my goodness. sorry. LGTM.

**Comment by icecrime** [comment]
LGTM

**Pull request merged**